### ISSUE TITLE
Gitlab backup to Scaleway Object Storage (S3) ?

### ISSUE THREAD

**Issue by SampicBE** [sameersbn/docker-gitlab#1979]
Hello,

I would like to know if it was possible to configure Gitlab backups towards a Scaleway Object Storage server.

Apparently their servers are S3 compatible, but I would like to know if one of you have already tried doing this ?

Thanks in advance.

**Comment by SampicBE** [comment]
up ? ....

**Comment by fire** [comment]
I recommend looking at the instructions for s3 using Google. Since s3 using Google is non-native, but uses the same theory.

**Comment by stephane-klein** [comment]
@SampicBE yes, you can configure GitLab Backup to Scaleway S3 with this configuration:

```
      - AWS_BACKUPS=true
      - AWS_BACKUP_REGION=fr-par
      - AWS_BACKUP_ENDPOINT=https://s3.fr-par.scw.cloud
      - AWS_BACKUP_ACCESS_KEY_ID=xxxxx
      - AWS_BACKUP_SECRET_ACCESS_KEY=xxxx
      - AWS_BACKUP_BUCKET=xxxx
```

Best regards,
Stéphane

**Comment by williamjie** [comment]
AWS_BACKUP_SIGNATURE_VERSION    S3  SIGNATURE config must

**Comment by D0wn3r** [comment]
> @SampicBE yes, you can configure GitLab Backup to Scaleway S3 with this configuration:
> 
> ```
>       - AWS_BACKUPS=true
>       - AWS_BACKUP_REGION=fr-par
>       - AWS_BACKUP_ENDPOINT=https://s3.fr-par.scw.cloud
>       - AWS_BACKUP_ACCESS_KEY_ID=xxxxx
>       - AWS_BACKUP_SECRET_ACCESS_KEY=xxxx
>       - AWS_BACKUP_BUCKET=xxxx
> ```
> 
> Best regards, Stéphane

Hi,
Where you put this type of config on an omnibus install on linux ?

Best regards
